samples/javascript_nodejs/12.nlp-with-luis/index.js

```diff
@@ -48,27 +44,47 @@ const luisApplication = {
     applicationId: luisConfig.appId,
     endpointKey: luisConfig.subscriptionKey,
     azureRegion: luisConfig.region
-}
+};
 
 // Create configuration for LuisRecognizer's runtime behavior.
 const luisPredictionOptions = {
     includeAllIntents: true,
     log: true,
     staging: false
-}
+};
 
 // Create adapter. See https://aka.ms/about-bot-adapter to learn more about adapters.
 const adapter = new BotFrameworkAdapter({
     appId: endpointConfig.appId || process.env.MicrosoftAppId,
     appPassword: endpointConfig.appPassword || process.env.MicrosoftAppPassword
 });
 
-// Create the bot that handles incoming Activities.
-const luisBot = new LuisBot(luisApplication, luisPredictionOptions);
+// Catch-all for errors.
+adapter.onTurnError = async (turnContext, error) => {
+    console.error(`\n [onTurnError]: ${ error }`);
```

Comment:
onTurnError nees to include load, save, clear pattern as follows:

    // Clear out state
    await conversationState.load(context);
    await conversationState.clear(context);
    // Save state changes.
    await conversationState.saveChanges(context);

Reply:
The LUIS, and QnA Maker samples (and their App Insights counterparts) don't use `ConversationState` at this time and are all single turn bots.